examples/features/retry/client/main.go

```diff
@@ -47,16 +47,19 @@ var (
 		}]}`
 )
 
-// use grpc.WithDefaultServiceConfig() to set service config
-func retryDial() (*grpc.ClientConn, error) {
+// newClientWithRetries uses grpc.WithDefaultServiceConfig() to set
+// service config, and creates the channel. However, the recommended
```

Comment:
A minor improvement:

However, the recommended approach is to fetch the retry configuration (which is part of the service config) from the name resolver rather than defining it on the client side.